pyspecProcScripts/slice_FID_from_echo.py

```diff
@@ -25,6 +25,7 @@ def slice_FID_from_echo(
     =======
     s:      phase corrected and sliced data
```

Comment:
again unsure why this is included in this PR

Reply:
you added this.  not sure why you're asking me :-)